src/main/java/at/gammastrahlung/monopoly_server/network/websocket/MonopolyMessageHandler.java

```diff
@@ -214,10 +231,7 @@ public static ServerMessage endGame(WebSocketPlayer player) {
      * @return ServerMessage wih the given type and a updateType matching the type of the updateObject
      */
     private static ServerMessage generateUpdateMessage(ServerMessage.MessageType messageType, Object updateObject) {
-        var message = ServerMessage.builder()
-                .type(messageType)
-                .messagePath("update")
-                .jsonData(gson.toJson(updateObject));
+        var message = ServerMessage.builder().type(messageType).messagePath("update").jsonData(gson.toJson(updateObject));
```

Comment:
please revert this, it's hard to read and not an improvement, makes retention worse for no reason